examples/features/authz/README.md

```diff
@@ -0,0 +1,16 @@
+# RBAC authorization
+
+This example uses the `StaticInterceptor` from the `google.golang.org/grpc/authz` package.
+It uses a header based RBAC policy to match each gRPC method to a required role. For simplicity,
+the context is injected with mock metadata which includes the required roles, but this should
+be fetched from an appropriate service based on the authenticated context.
+
+## Try it
+
+```
+go run server/main.go
+```
+
+```
+go run client/main.go
```

Comment:
Could we add a paragraph or two here to briefly explain what the client & server do, please?  E.g. "Server requires role X for Y; client sets X and tries Y, which succeeds, then sets Z and tries Y, which fails".

Thanks!

Reply:
Just pushed some extra details in the README, let me know what you think :)